en/pocket_privacy_policy.md

```diff
@@ -0,0 +1,127 @@
+# Privacy Policy
+
+## *Posted October 11, 2021*
+
+This notice describes the Privacy Policy that applies to all information collected or submitted when you install or use the Pocket application or any of the other proprietary software applications or services we provide, and when you visit our website (all of which are referred to collectively herein as the "Pocket Technologies"). The Pocket Technologies are owned and licensed by Mozilla Corporation (referred to herein as "we," "us," "our," "Mozilla"), and are operated by Mozilla's business unit known as "Pocket," which has developed applications and services for use in connection with a variety of devices and platforms, including without limitation, Windows, Mac, iPhone, iPad, Kindle Fire, Android, BlackBerry, Windows phone, Chrome, Safari, Firefox and other mobile devices and computers.
+
+We understand your concerns about privacy. We want you to understand what types of information we collect and what may happen to that information if you install the Pocket application, visit the Pocket Website or use any of the Pocket Technologies.
+
+Our Privacy Policy includes the following provisions, and you can jump to each by selecting the links below:
+
+* [User Accounts and Collection of Information](#accounts)
```

Comment:
You have links to anchors but no anchors? There should be something like `{: #accounts }` next to the reference title.

https://raw.githubusercontent.com/mozilla/legal-docs/main/en/firefox_privacy_notice.md

Reply:
Sounds good to me.